.github/workflows/cibuildwheel.yml

```diff
@@ -0,0 +1,116 @@
+name: Build and Upload Wheels
+
+on:
```

Comment:
Like `cvc5` this runs on release, builds on a cron job (once a month in this case), and supports manual workflow dispatch.